json/shared/src/main/scala/fs2/data/json/JsonContext.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2019 Lucas Satabin
```

Comment:
Super-minor: 2020?
I'm still in favor of having a sbt plugin handle these headers btw